File: MusicFestival/Domain/OfficeEmployee.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace MPP_MusicFestival.Domain
{
    public class OfficeEmployee : Entity<long>
    {
        public string firstName { get; set; }
        public string lastName { get; set; }
        public long CNP { get; set; }
        public string username { get; set; }
        public string password { get; set; }
        


        public OfficeEmployee(long id, string firstName, string lastName, long CNP, string username, string password) : base(id)
        {
            this.firstName = firstName;
            this.lastName = lastName;
            this.CNP = CNP;
            this.username = username;
            this.password = password;
        }

        public OfficeEmployee(string firstName, string lastName, long CNP, string username, string password) : base(0)
        {
            this.firstName = firstName;
            this.lastName = lastName;
            this.CNP = CNP;
            this.username = username;
            this.password = password;
        }

        public override string ToString()
        {
            return "Cod angajat:" + base.ToString() + " Nume:" + lastName + 
                " Prenume:" + firstName + " CNP:" + CNP;
        }
    }
}

File: MPP_MusicFestival_Java/FestivalModel/src/main/java/festival/domain/OfficeEmployee.java
package festival.domain;

import java.io.Serializable;

public class OfficeEmployee extends Entity<Long> implements Serializable {

    private String firstName;
    private String lastName;
    private Long CNP;
    private String username;
    private String password;


    public OfficeEmployee(Long id, String firstName, String lastName, Long CNP, String username, String password) {
        setId(id);
        this.firstName = firstName;
        this.lastName = lastName;
        this.CNP = CNP;
        this.username = username;
        this.password = password;
    }

    public OfficeEmployee(String firstName, String lastName, Long CNP, String username, String password) {
        setId(0L);
        this.firstName = firstName;
        this.lastName = lastName;
        this.CNP = CNP;
        this.username = username;
        this.password = password;
    }

    public Long getId(){
        return  super.getId();
    }

    public void setId(Long id) {
        super.setId(id);
    }


    public String getFirstName() {
        return firstName;
    }

    public void setFirstName(String firstName) {
        this.firstName = firstName;
    }

    public String getLastName() {
        return lastName;
    }

    public void setLastName(String lastName) {
        this.lastName = lastName;
    }

    public Long getCNP() {
        return CNP;
    }

    public void setCNP(Long CNP) {
        this.CNP = CNP;
    }


    public String getUsername() {
        return username;
    }

    public void setUsername(String username) {
        this.username = username;
    }

    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
        this.password = password;
    }

    @Override
    public String toString() {
        return "Cod angajat:" + super.toString() + " Nume:" + lastName +
                " Prenume:" + firstName + " CNP:" + CNP;
    }
}
